augly/video/augmenters/ffmpeg/speed.py

```diff
@@ -1,32 +1,39 @@
 #!/usr/bin/env python3
 # Copyright (c) Facebook, Inc. and its affiliates.
 
-from typing import Tuple, Dict
+from typing import List
 
-import ffmpeg  # @manual
-from augly.video.augmenters.ffmpeg.base_augmenter import BaseFFMPEGAugmenter
-from augly.video.helpers import has_audio_stream
-from ffmpeg.nodes import FilterableStream
+from augly.video.augmenters.ffmpeg.base_augmenter import BaseVidgearFFMPEGAugmenter
 
 
-class VideoAugmenterBySpeed(BaseFFMPEGAugmenter):
+class VideoAugmenterBySpeed(BaseVidgearFFMPEGAugmenter):
     def __init__(self, factor: float):
         assert factor > 0, "Factor must be greater than zero"
         self.factor = factor
 
-    def add_augmenter(
-        self, in_stream: FilterableStream, **kwargs
-    ) -> Tuple[FilterableStream, Dict]:
+    def get_command(self, video_path: str, output_path: str) -> List[str]:
         """
         Changes the speed of the video
 
-        @param in_stream: the FFMPEG object of the video
+        @param video_path: the path to the video to be augmented
 
-        @returns: a tuple containing the FFMPEG object with the augmentation
-            applied and a dictionary with any output arguments as necessary
+        @param output_path: the path in which the resulting video will be stored.
+            If not passed in, the original video file will be overwritten
+
+        @returns: a list of strings containing the CLI FFMPEG command for
+            the augmentation
         """
-        video = in_stream.video.setpts(f"{1/self.factor}*PTS")
-        if has_audio_stream(kwargs["video_path"]):
-            audio = in_stream.audio.filter_("atempo", self.factor)
-            video = ffmpeg.concat(video, audio, v=1, a=1)
-        return video, {}
+        command = [
+            "-y",
+            "-i",
+            video_path,
+            "-vf",
+            f"setpts={1/self.factor}*PTS",
+            "-filter:a",
+            f"atempo={self.factor}",
```

Comment:
do we know that this part of the command still works when there is no audio stream?

Reply:
Works when there is no audio stream, tested it with this small script that I created in AugLy's root directory 
```python
import augly.video.functional as af
af.remove_audio('/tmp/test-video-3.mp4','/Users/admin/res.mp4')
af.change_video_speed('/Users/admin/res.mp4',factor=0.5)
```